zephyr/Kconfig

```diff
@@ -0,0 +1,44 @@
+# Nordic WiFi driver for nRF52840 and nRF5340
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config WPA_SUPP
+    bool "WPA WPA_SUPP"
+    select EARLY_CONSOLE
+    select NET_SOCKETS
+    select NET_SOCKETS_POSIX_NAMES
+    select NET_SOCKETS_PACKET
+    select NET_MGMT_EVENT_INFO
```

Comment:
If you select `NET_MGMT_EVENT_INFO`, you should also select its dependencies, i. e. `NET_MGMT` and `NET_MGM_EVENT`.